framework/src/play/cache/Cache.java

```diff
@@ -63,17 +73,37 @@ public static void clear() {
 
   /** Initialize the cache system. */
   public static void init() {
-    if ("enabled".equals(Play.configuration.getProperty("memcached", "disabled"))) {
+    Class<?> memcacheImplClass = null;
+    try {
+      memcacheImplClass = Class.forName("play.cache.MemcachedImpl");
```

Comment:
Nowadays, we usually implement such things using "Service Loader" mechanism (via file `META-INF/services`). 

Then this code would look simpler, something like

> CacheImpl impl = ServiceLoader.get("org.replay.Cache");
> // find any implementation of org.replay.Cache interface in all JAR files

See https://www.youtube.com/watch?v=-KGtZoFVzr8&t=11m28s

But ok, in case of Replay it's not too important. :)

Reply:
I knew about it but never used it. I just picked --as mentioned-- the same way of implementing that was used by the dependency-pluggable webserver that RePlay ships (nett3/4/javanet).

Thanks for pointing it out.